osquery/database/plugins/ephemeral.cpp

```diff
@@ -30,6 +58,24 @@ Status EphemeralDatabasePlugin::put(const std::string& domain,
   return Status(0);
 }
 
+Status EphemeralDatabasePlugin::put(const std::string& domain,
+                                    const std::string& key,
+                                    int value) {
+  db_[domain][key] = value;
+  return Status(0);
+}
+
+void EphemeralDatabasePlugin::dumpDatabase() const {
+  for (const auto& domainValue : db_) {
+    const auto& domain = domainValue.first;
+    for (const auto& keyValue : domainValue.second) {
+      const auto& key = keyValue.first;
+      const auto& value = keyValue.second;
+      std::cout << domain << "[" << key << "]: " << value << std::endl;
```

Comment:
I'd suggest to use just `'\n'` here instead of `std::endl`. There is no apparent reason to flush stdout buffer for every line, right?

Reply:
Yup, that's right. Not sure if the dump is the place for optimizations. std::endl just looks nicer to me :(

Reply:
You should be using the logging channels, not std::cout